### ISSUE TITLE
Keyboard navigation from PDF reader to context menu split button doesn't work

### ISSUE THREAD

**Issue by mrtcode** [zotero/zotero#3265]

**Comment by mrtcode** [comment]
https://forums.zotero.org/discussion/107530/zotero-7-strange-focusing-behaviour-in-ui

**Comment by mrtcode** [comment]
No longer relevant after UI redesign.
